src/auslib/AUS.py

```diff
@@ -73,29 +73,35 @@ def evaluateRules(self, updateQuery, transaction=None):
         self.log.debug("Looking for rules that apply to:")
         self.log.debug(updateQuery)
 
+        eval_metadata = {}
+
         if self.updates_are_disabled(updateQuery["product"], updateQuery["channel"], transaction) or self.updates_are_disabled(
             updateQuery["product"], getFallbackChannel(updateQuery["channel"]), transaction
         ):
             log_message = "Updates are disabled for {}/{}.".format(updateQuery["product"], updateQuery["channel"])
             self.log.debug(log_message)
-            return None, None
+            return None, None, eval_metadata
 
         rules = dbo.rules.getRulesMatchingQuery(updateQuery, fallbackChannel=getFallbackChannel(updateQuery["channel"]), transaction=transaction)
 
         # TODO: throw any N->N update rules and keep the highest priority remaining one?
         if len(rules) < 1:
-            return None, None
+            return None, None, eval_metadata
 
         rules = sorted(rules, key=lambda rule: rule["priority"], reverse=True)
         rule = rules[0]
+
+        eval_metadata["rule"] = rule["rule_id"]
```

Comment:
Let's use `rule_id` for the key here (rule already has a meaning, let's not overload it).